source/manuals/programming-languages/js.html.md.erb

```diff
@@ -23,23 +23,37 @@ review_in: 3 months
 
 ## Linting
 
-We follow [standardjs](http://standardjs.com/), an opinionated JavaScript linter.
+We follow [standardjs](http://standardjs.com/), an opinionated JavaScript linter. In cases where standard conflicts with a services browser support we use [standardx][].
```

Comment:
Could this part include a link to part of this page which explains why standardx is being used? I was thinking about the "Using standardx" section, as this bit could be a bit clearer about which rules should be overridden.

Reply:
Thanks for the suggestion Ian. I've reworked the "Using standardx" section to be more of a "When to use standardx" to try increase this clarity. How does that look to you?